manifests/k/KDE/Kate/21.08.2/KDE.Kate.installer.yaml

```diff
@@ -1,14 +1,12 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
-PackageIdentifier: KDE.Kate
-PackageVersion: 21.08.2
-MinimumOSVersion: 10.0.0.0
-InstallerType: nullsoft
-Scope: machine
+PackageIdentifier: KDE.Kate.Release
+PackageVersion: master-1459
+InstallerType: appx
 UpgradeBehavior: install
 Installers:
 - Architecture: x64
-  InstallerUrl: https://binary-factory.kde.org/view/Windows%2064-bit/job/Kate_Release_win64/1456/artifact/kate-21.08.2-1456-windows-msvc2019_64-cl.exe
-  InstallerSha256: D95F20BE05A06358090201A956D608DD0457CC7916A159403F83E671D7F045A7
+  InstallerUrl: https://binary-factory.kde.org/view/Windows%2064-bit/job/Kate_Release_win64/lastSuccessfulBuild/artifact/kate-21.08.2-1459-windows-msvc2019_64-cl-sideload.appx
```

Comment:
```suggestion
  InstallerUrl: https://binary-factory.kde.org/view/Windows%2064-bit/job/Kate_Release_win64/1459/artifact/kate-21.08.2-1459-windows-msvc2019_64-cl-sideload.appx
```

Reply:
`lastSuccessfulBuild` in the URL is only valid for 1 day (24 hours), compared to having the build number in the URL which would last for about 4 or 5 days, before needing to be replaced.

If someone doesn't regularly update this like KDE Connect, people will be unable to download due to the expired URL. There will also likely be issue spam from @wingetbot when the URLs are unavailable if it's not updated in a timely matter.

Reply:
> http://github.com/microsoft/winget-pkgs/pull/33731/commits/20eac3c65230bd500e863f243cf4183dd4853f97#r742086022.

Do observe "http://github.com/microsoft/winget-pkgs/pull/33718#discussion_r742010117" for explanation of why I am believing that this modification is advantageous.

Reply:
> the logic that is required for automatic replacement by new versions is much more simple

I don't see how this would make it easier to update the application's package because the URL is not static. It still requires modification.

> the automation that is currently present within this repository should remediate that problem.

Sure, but what if the automation is temporarily offline for maintenance which could also happen at any time.